src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/CcdCallbackController.java

```diff
@@ -27,26 +28,18 @@ public CcdCallbackController(AttachCaseCallbackService attachCaseCallbackService
         this.attachCaseCallbackService = attachCaseCallbackService;
     }
 
-    @PostMapping(
-        path = "/callback/attach_case",
-        consumes = APPLICATION_JSON_VALUE,
-        produces = APPLICATION_JSON_VALUE
-    )
-    public ResponseEntity<CallbackResponse> handleCallback(@RequestBody CallbackRequest callback) {
+    @PostMapping(path = "/attach_case")
+    public ResponseEntity<CallbackResponse> attachToCase(@RequestBody CallbackRequest callback) {
```

Comment:
Perhaps wrapping of `ResponseEntity` is not needed at all?

Reply:
Good point!
Updated.